tasks/kernel_matrix_testing/compiler.py

```diff
@@ -74,17 +74,25 @@ def image(self):
 
         return f"{image_base}_{self.arch.ci_arch}{suffix}:{version}"
 
-    @property
-    def is_running(self):
+    def _check_container_exists(self, allow_stopped=False):
         if self.ctx.config.run["dry"]:
             warn(f"[!] Dry run, not checking if compiler {self.name} is running")
             return True
 
-        res = self.ctx.run(f"docker ps -qf \"name={self.name}\"", hide=True)
+        args = "-a" if allow_stopped else ""
```

Comment:
You already have the `-`
```suggestion
        args = "a" if allow_stopped else ""
```